routing/ROUTING_V1_HTTP.md

```diff
@@ -90,6 +90,18 @@ Response limit: 100 providers
 
 Each object in the `Providers` list is a *read provider record*.
 
+#### List of Routers
+
+A list of routers that should be consulted for responses. The specification imposes no constraints on the order by which the results are returned. Implementers are free to return results as they are found.
+
+If none are provided the backend may decide which ones to use. For a list of known routers see below.
+
+To select all routers use `?routers=*`. To determine the list of available routers see the `OPTIONS` API below
+
+### `OPTIONS /routing/v1/providers`
```

Comment:
For the UX sake, if we introduce new endpoint, we should make it plain `GET`, so the URL can be loaded in regular browser or curl.